meta_update.py

```diff
@@ -150,7 +151,10 @@ def main():
         if args.newversion is not None:
             manipulator.version(args.newversion)
         manipulator.update_authors()
-        manipulator.save()
+        filename = None
+        if args.outdir is not None:
+            filename = f'{args.outdir}/{manipulator.default_filename}'
```

Comment:
I would prefer a `pathlib.Path` based approach for this. Yes, not needed for us. But still nicer?